aioshelly/rpc_device/device.py

```diff
@@ -171,7 +181,7 @@ async def initialize(self, async_init: bool = False) -> None:
             self.initialized = True
         except InvalidAuthError as err:
             self._last_error = InvalidAuthError(err)
-            _LOGGER.debug("host %s: error: %r", ip, self._last_error)
+            _LOGGER.debug("host %s:%s: error: %r", ip, self.port, self._last_error)
```

Comment:
I think we should be consistent here, either we use local variables `ip` and `port` or `self.ip_address` and `self.port` everywhere.